src/server/identity/server/dex_web_test.go

```diff
@@ -119,18 +120,19 @@ func TestUpdateIDP(t *testing.T) {
 	err := env.GetDexDB().CreateConnector(dex_storage.Connector{ID: "conn", Type: "github", Config: []byte(`{"clientID": "test1", "redirectURI": "/callback"}`)})
 	require.NoError(t, err)
 
-	// Create an auth request that hasn't done the flow
-	require.NoError(t, env.GetDexDB().CreateAuthRequest(dex_storage.AuthRequest{
-		ID:       "testreq",
-		ClientID: "testclient",
+	require.NoError(t, env.GetDexDB().CreateClient(dex_storage.Client{
+		ID:           "testclient",
+		RedirectURIs: []string{"http://example.com/callback"},
 	}))
 
 	// make a request to authenticate
 	recorder := httptest.NewRecorder()
-	req := httptest.NewRequest("GET", "/auth/conn?req=testreq", nil)
+	req := httptest.NewRequest("GET", "/auth/conn?req=testreq&client_id=testclient&redirect_uri=http%3A%2F%2Fexample.com%2Fcallback&scope=openid&response_type=code", nil)
```

Comment:
Dex requires all these parameters to be passed to the connector, previously I think they were just stashed in the auth request. I haven't dug into the architectural reason for the change but it should be transparent to end users, we just need to handle it when we're trying to programmaticaly drive the OAuth login flow.